README.md

```diff
@@ -1,89 +1,15 @@
 # [Codurance Web Site](https://codurance.com/)
 
-This is the source of the Codurance web site. It is built and pushed to the [*gh-pages*](https://github.com/codurance/site/tree/gh-pages) branch.
-
-## Initialisation
-
-### Installing on the MacOSX / Linux
-
-Note: you might have to [update Ruby to 2.3.1](http://stackoverflow.com/a/33883667) first.
-
-Run the below commands to install the necessary gems (you might need to prefix the commands with `sudo`):
-
-```
-gem install bundler
-gem install rspec-core -v '3.4.1'
-gem install jekyll
-bundle install
-```
-
-In case the `bundle install` command aborts, with a message about a missing gem, install the missing gem indicated in the message and re-run `bundle install`. Repeat this for the number of times missing gems are notified.
-
-Note: this should take a bit of time to install the gems.
-
-If you are using RVM, you should create a *gh-pages* gemset first using the command `rvm gemset create gh-pages`, which will help you isolate your gems from your standard Ruby installation; you should then run `rvm gemset use gh-pages` when you start a new terminal session.
-
-In case of other problems, refer to the [troubleshooting section](#troubleshooting-section).
-
-## Building
-
-To build the site, run `rake build` or `jekyll serve` from the root directory of the project. You will not often have to do this manually; more often, you'll want a server in the background so you can verify your changes are as expected.
-
-Use script file `buildAndRunSite.sh` to achieve the above.
-
-## Serving the Web Site Locally
-
-Running `rake serve` will serve the web site at [http://localhost:4000/](http://localhost:4000/) and watch for incremental changes in the background so you can test your site. If you want your page generated quicker then run `rake servequick` which will only generate the most recent 3 blogs and the rest of the site.
-
-The above commands should be executed from the root directory of the project.
-
-## Serving the Web Site from Docker
+# Setup the <del>development</del> writing environment 
 
 **[Docker Installation required]**
 
-To build and serve the site from Docker you need to build a local image based.
-Dockerfile for is ready to use in directory `docker`.
-
 You need to execute the following command to prepare an image and run a container:
 
 ```
-docker build --tag codurance-site:local -f ./docker/Dockerfile-local ./docker/
-docker run -d -v <<root_directory_of_this_project>>:/site -p 4000:4000 --name codurance-site codurance-site:local
+docker build --tag codurance-site:local -f ./Dockerfile-local . # you need to do this only once - this builds the container with all the dependencies for running the site
```

Comment:
What about moving the comments above the command? It would make it easier to see the separation

```
# you need to do this only once - this builds the container with all the dependencies for running the site
docker build --tag codurance-site:local -f ./Dockerfile-local .

# you need to run this every time you want to stand up the local server - it should watch for changes in the local files automatically though
docker run -i -t -v `pwd`:/site -p 4000:4000 codurance-site:local 
```